app/src/main/java/waiter/App.java

```diff
@@ -15,7 +15,7 @@ public static void main(String[] args) {
 
         int port = 5000;
         Listener listener = new Listener();
-        Messenger messenger = new Messenger(new HttProtocol(new RequestParser(), new Router(), new ResponseBuilder()));
+        Messenger messenger = new Messenger(new HttProtocol(new RequestParser(), new Router(), new ResponseFormatter()));
```

Comment:
Nitpick, consider break these into separate lines:
```java
HttpProtocol protocol = new HttProtocol(
  new RequestParser(),
  new Router(),
  new ResponseFormatter()
);

Messenger messenger = new Messenger(protocol);
```

Reply:
Addressed in [this commit](https://github.com/jonmanahan/waiter/pull/17/commits/ea062f7676d20cddf0fc85eb10395b89a9365b88)